### ISSUE TITLE
Rename Request, Response Decorators to Req and Res

### ISSUE THREAD

**Issue by Kaizodo** [serhiisol/node-decorators#116]
Hello, 

this is not a issue but a feature request , i am using express decorators in my project but one thing is really a big frustrating for me is  i cannot typecast Request decorator variable as express Request type ,i also don't want it to set Any as well, so to solve this problem i have to do  code something like below 

```typescript
import { Body, Controller, Post, Request as Req, Response as Res } from "@decorators/express";
import { Request, Response } from "express";

  @Post("/detail")
    async detail(@Req() req: Request, @Res() res: Response) {
      res.send("works");
    }
```

as you can see i have to put little extra time for the imports to rename Request to Req and Response to Res, i really do not like to use ANY if there is a type available for param. also in NEST.js framework they have named Request to Req and Response to Res.

**Comment by angelxmoreno** [comment]
I like this feature request. I am coming from routing-controllers

**Comment by serhiisol** [comment]
This is will be a breaking change to those, who are using library. Don't you like the idea of aliases?

**Comment by Kaizodo** [comment]
instead of renaming them can you add @Req and @Res decorators as alternates ? with this way both full version and short version will be available and it will not break any existing user's code base.

**Comment by vikicoder007** [comment]
Any update on this ?, i really like the idea of @Req and @Res decorators , also if would be possible to send response on return like nestjs , we just have to return object or some predefined functions in framework to send response

**Comment by serhiisol** [comment]
merged in #114

**Comment by serhiisol** [comment]
Will release it shortly.
